### ISSUE TITLE
automatically disable existing deliveries that have a frequency that's more than hourly

### ISSUE THREAD

**Issue by rephus** [lightdash/lightdash#10669]
- automatically disable existing deliveries that have a frequency that's more than hourly.

Depends on https://github.com/lightdash/lightdash/issues/10319

_Originally posted by @rephus in https://github.com/lightdash/lightdash/issues/10432#issuecomment-2219948106_

**Comment by rephus** [comment]
We have a simple UI to set deliveries by hour/day/week. But we also allow custom cron strings. You can type anything in there `* * * * *` or `0/5 * * * *` . 

I've already limited this option in the UI and the backend to now allow more than hourly. 

This PR is to `disable` existing schedulers, after we implement https://github.com/lightdash/lightdash/issues/10319

**Comment by rephus** [comment]
tl;dr don't do this on pro customers
See last comment from dependency https://github.com/lightdash/lightdash/issues/10319#issuecomment-2222609890

**Comment by rephus** [comment]
Update: There are no active schedulers with invalid frequencies in shared cloud instances, so we can close this ticket
